tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js

```diff
@@ -207,6 +207,39 @@ test.describe( 'Nested Tabs tests', () => {
 		await expect( activeTab ).toHaveCSS( 'margin-bottom', '50px' );
 		await expect( lastTab ).toHaveCSS( 'margin-top', '25px' );
 	} );
+
+	test( 'Check that active and non-active tabs color changes on hover', async ( { page }, testInfo ) => {
+		// Arrange.
+		const wpAdmin = new WpAdminPage( page, testInfo );
+
+		await wpAdmin.setExperiments( {
```

Comment:
@davseve Comment from Tzvi: can we set the experiments to inactive in the end. 
Addition from me: you can also create `afterAll()` and `beforeAll()` functions.

Reply:
Done, 
Regarding the `afterAll()` and `beforeAll()`, we can add it in the next task.